sdk/analytics/src/main/java/com/klaviyo/analytics/model/BaseModel.kt

```diff
@@ -1,4 +1,4 @@
-package com.klaviyo.coresdk.model
+package com.klaviyo.analytics.model
```

Comment:
Could be in `core` module maybe? Since its pretty abstract

Reply:
True, just gotta double check that it won't complain when we expose `Profile` publicly, but I don't think it will.

Reply:
Done